File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/incoder-1/find_in_sorted.py
--- 
+++ 
@@ -6,7 +6,7 @@
         if x < arr[mid]:
             return binsearch(start, mid)
         elif x > arr[mid]:
-            return binsearch(mid, end)
+            return binsearch(mid+1, end)
         else:
             return mid
 

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/gpt-neo-1.3/find_in_sorted.java
--- 
+++ 
@@ -6,7 +6,7 @@
     if (x < arr[mid]) {
         return binsearch(arr, x, start, mid);
     } else if (x > arr[mid]) {
-        return binsearch(arr, x, mid, end);
+        return binsearch(arr, x, mid + 1, end);
     } else {
         return mid;
     }
